build/karma.conf.babel.js

```diff
@@ -35,6 +38,9 @@ module.exports = (karmaConfig) => {
           ...webpackConfig.module.loaders,
         ],
       },
+      plugins: [
+        new webpack.DefinePlugin(config.compiler_globals),
+      ],
```

Comment:
Recommend using `process.env.NODE_ENV`. It's more verbose, but is expected in webpack builds nowadays (esp. for React), whereas the former is just an internal helper. This way users won't have to have `__PROD__` as a global.